There seems to be a missing include in these files. Without them, the build
fails on aarch64-darwin with errors like:

error: call to undeclared function 'qemu_system_reset_request'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
        qemu_system_reset_request();

diff --git a/hw/arm/stm32f2xx.c b/hw/arm/stm32f2xx.c
index d8397d954..2d43fd288 100644
--- a/hw/arm/stm32f2xx.c
+++ b/hw/arm/stm32f2xx.c
@@ -26,6 +26,7 @@
 #include "hw/ssi.h"
 #include "hw/block/flash.h"
 #include "sysemu/blockdev.h" // drive_get
+#include "sysemu/sysemu.h"
 
 static const char *stm32f2xx_periph_name_arr[] = {
     ENUM_STRING(STM32_UART1),
diff --git a/hw/arm/stm32f4xx.c b/hw/arm/stm32f4xx.c
index 711cc7c88..28ca9ed5d 100644
--- a/hw/arm/stm32f4xx.c
+++ b/hw/arm/stm32f4xx.c
@@ -26,6 +26,7 @@
 #include "hw/ssi.h"
 #include "hw/block/flash.h"
 #include "sysemu/blockdev.h" // drive_get
+#include "sysemu/sysemu.h"
 
 static const char *stm32f4xx_periph_name_arr[] = {
     ENUM_STRING(STM32_UART1),
diff --git a/hw/arm/stm32f7xx.c b/hw/arm/stm32f7xx.c
index 5e5ccc8fb..adf68ad04 100644
--- a/hw/arm/stm32f7xx.c
+++ b/hw/arm/stm32f7xx.c
@@ -28,6 +28,7 @@
 #include "hw/ssi.h"
 #include "hw/block/flash.h"
 #include "sysemu/blockdev.h" // drive_get
+#include "sysemu/sysemu.h"
 
 static const char *stm32f7xx_periph_name_arr[] = {
     ENUM_STRING(STM32_UART1),
